daily_milestone.go

```diff
@@ -225,12 +244,11 @@ func main() {
 		logger.Println(err)
 	}
 
-	newMilestones := createMilestoneData(advance)
-
+	newMilestones := createMilestoneData(advance, strings.ToLower(milestoneTime))
 	for i, newPair := range newMilestones {
 		for _, oldPair := range oldMilestones {
 			if oldPair.Title == newPair.Title {
-				newMilestones = append(newMilestones[:i], newMilestones[(i+1):]...)
+				newMilestones = append(newMilestones[:i], newMilestones[(i):]...)
```

Comment:
What was the reasoning for this change?

Reply:
I couldn't understand it was not iterating properly with weekly and monthly. So played around and this worked. Not too sure why.

Reply:
This changes the outcome and doesn't implement a drop as we want. It actually wouldn't drop any milestone:
https://play.golang.org/p/tvTQyre7Rj